common/recordings.md

```diff
@@ -139,3 +140,39 @@ If a storage destination is not configured for recordings and if you choose to r
 | Stream ID   | Unique identifier for a particular stream of a room (audio-video/screenshare)                                                                                                               |
 | Track ID    | Unique identifier for a particular track (audio or video) of a stream                                                                                                                       |
 | Layer Index | Layer index values show descending HLS resolutions - 0(1080p), 1(720p), 2(480p), 3(360p) and 4(240p). If highest resolution of template is 720p, then 0(720p), 1(480p), 2(360p) and 3(240p) |
+
+## Chat Recording
+
+Chat recording is a feature through which you will receive all chats messages sent by peers during the SFU/browser recording. Chat recording is available for both SFU recording and browser recording. Only public chats sent to all roles will be recorded. The `.csv` file will be uploaded to the recording bucket configured for your video recordings. Chat recording is currently enabled from backend for few customers. The file header will be: `SenderPeerID,SenderName,SenderUserID,Roles,SentAt,Type,Message`
+
+**Header information**
+
+| Header | Description |
+| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| SenderPeerID |Sender's peer id |
+| SenderName | Sender's name |
+| SenderUserID | Sender's user id |
+| Roles | Roles array to which the message is sent `[]` in case sent to all roles |
+| SentAt| SentAt in RFC.3339 format |
```

Comment:
is it the same as ISO-8601? `2020-12-09T16:09:53Z`?

Reply:
yes.

Reply:
awesome. can we please add an example to this? would prevent a google lookup on what is this RFC

Reply:
ok saw that it has been merged. lgtm, will do this later